.yarn/patches/@babel-helper-compilation-targets-npm-7.22.5-5e6d9af186.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/lib/options.js b/lib/options.js
```

Comment:
We can remove this patch once `@babel/helper-compilation-targets` is published and `@babel/helper-define-polyfill-provider` updates its dependency.